### ISSUE TITLE
emacs creates synthetic bold ("emboldened") variant via overstriking

### ISSUE THREAD

**Issue by real-or-random** [rainstormstudio/nerd-icons.el#65]
I'd like to raise your awareness to https://debbugs.gnu.org/cgi/bugreport.cgi?bug=67810, which I have reported to emacs.

tl;dr: emacs creates a bold version of the nerd icons whenever bold is requested as a face attribute.  This is undesirable because the resulting icon may be too wide and also because it just looks bad. 

But unless upstream does something about it, I don't think there's a lot that can be done here:
 - Set `:weight 'regular` in all faces here. This won't hurt, but it solves only part of the problem (see [my comments](https://debbugs.gnu.org/cgi/bugreport.cgi?bug=67810#41) in the emacs bug).
 - Include [the fontconfig snippet](https://debbugs.gnu.org/cgi/bugreport.cgi?bug=67810#47) in the README. That also won't hurt, but it only works on platforms with fontconfig, and the user needs to do it manually.

Alternatively, this package could also install a patched font that claims to be bold but is just the regular variant. That's a bit ugly. But at least it would just solve the problem.
